front/docker/Dockerfile.playwright-ci

```diff
@@ -0,0 +1,9 @@
+ARG PLAYWRIGHT_VERSION=latest
```

Comment:
Since this is a ci only file, I'd rather it lives in the /docker/ folder (to avoid pestering the front maintainers)

Reply:
Sure